layers/meta-resin-allwinner/recipes-kernel/xradio/xradio.bb

```diff
@@ -0,0 +1,17 @@
+
+SUMMARY = "Xradio WiFi driver for orangepi-zero (xr819)"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=a23a74b3f4caf9616230789d94217acb"
+
+inherit module
+
+SRCREV = "b7789c274091344eeff47cf34a48e0b225de852b"
+
+SRC_URI = "git://github.com/fifteenhex/xradio.git;protocol=https \
+          "
+
+S = "${WORKDIR}/git"
+
+EXTRA_OEMAKE += "-C ${STAGING_KERNEL_DIR} M=${S}"
+
+KERNEL_MODULE_AUTOLOAD += "xradio_wlan"
```

Comment:
I see this recipe is in the thud branch of meta-sunxi. Can you remove it from here?
We will have thud support shortly, it will be best if we do not duplicate it here also

Reply:
Yes I can do that. Will you link the commit for updating the submodule in this pr?